docs/beta/tutorial/get_version.md

```diff
@@ -0,0 +1,51 @@
+# Tutorial: Get Tamr version
+This tutorial will introduce basic usage of the Tamr Python client.  It will guide you through the process of configuring the connection to a Tamr instance and retrieving the version of that instance.
+
+## Preparation
+To complete this tutorial you will need access to a Tamr instance.  Specifically you should have a username and password that allow you to log in to Tamr, as well as the socket address of the instance.  The socket address is composed of
+1. The protocol, such as `"https"` or `"http"`
+2. The host, which may be `"localhost"` if the instance is deployed from the same machine from which your Python code will be run
+3. The port at which you access the Tamr user interface, typically `9100`
+
+When you view the Tamr user interface in a browser, the url is `<protocol>://<host>:<port>`.
+
+## Procedure
+### The Session Object
+The Tamr Python client uses `Session` objects primarily to persist the user's authentication details across requests made to the server where Tamr is hosted.
+
+A `Session` object carries authentication credentials derived from a username and password, and is not explicitly tied to any single Tamr instance. For more details, see the documentation for the [Requests library](https://requests.readthedocs.io/en/master/user/advanced/#session-objects).
+
+ - Use your username and password to create an instance of `tamr_client.UsernamePasswordAuth`.
+ - Use the function `tamr_client.session.from.auth` to create a `Session` object.
+```python
+import tamr_client as tc
+import os
+
+username = os.environ['TAMR_USERNAME']
+password = os.environ['TAMR_PASSWORD']
+
+auth = tc.UsernamePasswordAuth(username, password)
+session = tc.session.from_auth(auth)
+```
+### The Instance Object
+An `Instance` object models the installation or instance of Tamr with which a user interacts via the Python client.
+
+- Create an `Instance` object using the `protocol`, `host`, and `port` of your Tamr instance.
+```python
+protocol = "http"
+host = "localhost"
+port = 9100
+
+instance = tc.Instance(protocol=protocol, host=host, port=port)
+```
+### Getting the version of Tamr
+With the `Session` and `Instance` objects defined, you can now interact with the API of the Tamr instance.  One simple example is fetching the version of the Tamr software running on the server.
+
+- Use the function `tc.version` and print the returned value.
+
+```python
+print(tc.version(session, instance))
+```
+If successful, the result should be something similar to `v2020.016.0`.
+ 
+Congratulations! This is just the start of what can be done with the Tamr Python client.  Please see other tutorials and examples, or explore on your own.
```

Comment:
nit: best practice for copy is to avoid the word "please" and instead explain what the user will gain if they do something.

```suggestion
Congratulations! This is just the start of what can be done with the Tamr Python client.

To continue learning, see other tutorials and examples.
```